ClemBot.Bot/bot/bot_secrets.py

```diff
@@ -244,6 +258,7 @@ def load_production_secrets(self) -> None:
         self.api_key = os.environ.get("API_KEY")  # type: ignore
         self.site_url = os.environ.get("SITE_URL")  # type: ignore
         self.docs_url = os.environ.get("DOCS_URL")  # type: ignore
+        self.allow_bot_input_ids = os.environ.get("ALLOW_BOT_INPUT_IDS")  # type: ignore
```

Comment:
This wont work, it has to insert a list of ints. this inserts a string, see line 248 for example